src/main/java/com/csselect/database/mysql/MysqlDatabaseAdapter.java

```diff
@@ -0,0 +1,424 @@
+            return null;
+        }
+        MysqlOrganiserAdapter adapter;
+        try {
+            adapter = new MysqlOrganiserAdapter();
+        } catch (SQLException e) {
+            e.printStackTrace();
+            return null;
+        }
+        adapter.setEmail(email);
+        adapter.setPassword(hash, salt);
+        return new Organiser(adapter);
+    }
+
+    @Override
+    public void registerGame(Organiser organiser, Game game) {
+        if (!gameMap.containsKey(game)) {
+            gameMap.put(game, organiser);
+            try {
+                executeMysqlUpdate("UPDATE organiser_id SET " + organiser.getId() + " FROM games WHERE"
+                        + " (id='" + game.getId() + "');");
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        }
+    }
+
+    @Override
+    public void removeGame(Game game) {
+        gameMap.remove(game);
+        try {
+            executeMysqlUpdate("DELETE FROM games WHERE (id='" + game.getId() + "');");
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Executes the given Mysql-query on the main database
+     * @param query query to execute
+     * @return ResultSet of the operation
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    ResultSet executeMysqlQuery(@Language("sql") String query) throws SQLException {
+        return executeMysqlQuery(query, PRODUCT_DATABASE_NAME);
+    }
+
+    /**
+     * Executes the given Mysql-query on the given database
+     * @param query query to execute
+     * @param databaseName database to execute the query on
+     * @return ResultSet of the operation
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    ResultSet executeMysqlQuery(@Language("sql") String query, String databaseName) throws SQLException {
+        MysqlDataSource dataSource = dataSources.getOrDefault(databaseName, createDataSource(databaseName));
+        Connection connection = dataSource.getConnection();
+        Statement statement = connection.createStatement();
+        return statement.executeQuery(query);
+    }
+
+    /**
+     * Executes the given Mysql-Update on the main database
+     * @param update update query to execute
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    void executeMysqlUpdate(@Language("sql") String update) throws SQLException {
+        executeMysqlUpdate(update, PRODUCT_DATABASE_NAME);
+    }
+
+    /**
+     * Executes the given Mysql-Update on the given database
+     * @param update update query to execute
+     * @param databaseName database to execute the query on
+     * @throws SQLException Thrown when there is an error executing the given statement
+     */
+    void executeMysqlUpdate(@Language("sql") String update, String databaseName) throws SQLException {
```

Comment:
maybe use prepared statments instead of just passing user input strings as sql commands